upt_macports/upt_macports.py

```diff
@@ -88,13 +103,15 @@ def _pkgname(self):
 
     @staticmethod
     def _normalized_macports_name(name):
-        name = name # for perl changing to lower case may cause error
+        name = name  # for perl changing to lower case may cause error
         return name
 
+
 class MacPortsRubyPackage(MacPortsPackage):
     template = 'ruby.Portfile'
 
-class MacportsBackend(upt.Backend):
+
+class MacPortsBackend(upt.Backend):
```

Comment:
if you change the capitalization here (I do agree with that BTW), then I think you'll have to do that as well in in https://github.com/macports/upt-macports/blob/20b55dcc6a06f48da8a22c415bc74315dda5cb2e/setup.cfg#L26

Reply:
You're right about that. I missed this by the way :)

This is also why I'd rather see smaller "self-contained" PRs: they are easier to review in github.

Reply:
So I guess instead of doing different PR from next time, seeing that few of the current commits can be merged and few not.
If everyone agrees then I'll close this PR and open up different PRs for different purposes namely:
- Add Travis support
- Add license conversion
- Add tests

Reply:
My 2 cents. I agree that it generally makes more sense to do small and self-contained pull requests on independent code, so if one thing is ok, it can be merged immediately, and the other one can be fixed / discussed later.

But on the other hand you started this PR to do one feature, and then it turned out that some more stuff needs to be done to consider this feature complete. If you write ten lines per day this can be done, but if one produces hundreds of lines of code per day with dependencies (feature A needs to be merged before feature B), it becomes a mess to deal with a zillion pull requests, and I see no reason to be overly draconic about what goes in what pull request, and at the end produce 10 times less code than if allowed to work efficiently.

From my point of view it makes more sense to just clean the commits and not open new pull requests for its own sake.

Reply:
@mojca @Korusuke yes, that's what I said - finish up this PR with separating commits into logical changes for now and move on ;) Let's just keep this discussion in mind for the future and be flexible if a single-purpose PR evolves into something more!

Reply:
Okeyy then I will just separate all out into different commits and push the changes soon.